uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-requests.js

```diff
@@ -249,7 +249,7 @@ define([
     function loadAbsencePeriods() {
       return AbsencePeriod.all()
         .then(function (absencePeriods) {
-          vm.absencePeriods = absencePeriods;
+          vm.absencePeriods = _.sortBy(absencePeriods, 'title');;
```

Comment:
You can't sort by the title, a period can have any arbitrary title. You need to use the start date